components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -576,6 +576,43 @@ public int getBusinessOwnersCount(int tenantId) throws AppManagementException {
     }
 
     /**
+     * Search business owners.
```

Comment:
Formatting and Java doc comments issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files